### ISSUE TITLE
SFT test dataset

### ISSUE THREAD

**Issue by CarlHuangNuc** [THUDM/WebRL#14]
Which dataset should we use for validation or test ？

**Comment by CarlHuangNuc** [comment]
![image](https://github.com/user-attachments/assets/330d5b29-b287-4d98-a479-0a00e7ea21dc)
we do not use any test dataset.

**Comment by QZH-777** [comment]
> ![image](https://private-user-images.githubusercontent.com/104048874/390988468-330d5b29-b287-4d98-a479-0a00e7ea21dc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMxNjAxOTksIm5iZiI6MTczMzE1OTg5OSwicGF0aCI6Ii8xMDQwNDg4NzQvMzkwOTg4NDY4LTMzMGQ1YjI5LWIyODctNGQ5OC1hNDc5LTBhMDBlN2VhMjFkYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwMlQxNzE4MTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01ZGEzNWJmYTk2ODgxODQ3Y2ViOWViYzE4NGQ4ODA4OTYxZGQwMTE1Njk0OTg5ODkyYjZkZGU3NjVjZmZlZTg1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.v5nW-gBbt4WO6ujagNGENur2y3miHVsYIhk7nKhniGI) we do not use any test dataset.

You can create a validation set by splitting the training dataset!
